.changes/unreleased/Added-20220315-093847.yaml

```diff
@@ -0,0 +1,5 @@
+kind: Added
+body: New --no-prereleases option to changie latest
```

Comment:
No need to say "New" it is implied with it being in the release notes and under the Added kind. See other changelogs for new options.

Reply:
ok done